torchmetrics/utilities/imports.py

```diff
@@ -68,11 +68,11 @@ def _compare_version(package: str, op: Callable, version: str) -> Optional[bool]
     return op(pkg_version, Version(version))
 
 
-_TORCH_LOWER_1_4 = _compare_version("torch", operator.lt, "1.4.0")
-_TORCH_LOWER_1_5 = _compare_version("torch", operator.lt, "1.5.0")
-_TORCH_LOWER_1_6 = _compare_version("torch", operator.lt, "1.6.0")
-_TORCH_GREATER_EQUAL_1_6 = _compare_version("torch", operator.ge, "1.6.0")
-_TORCH_GREATER_EQUAL_1_7 = _compare_version("torch", operator.ge, "1.7.0")
-_LIGHTNING_AVAILABLE = _module_available("pytorch_lightning")
-_TORCH_FIDELITY_AVAILABLE = _module_available("torch_fidelity")
-_SCIPY_AVAILABLE = _module_available("scipy")
+_TORCH_LOWER_1_4: Optional[bool] = _compare_version("torch", operator.lt, "1.4.0")
```

Comment:
Given that we don't really work without torch, having this as Optional feels weird

Reply:
this comes from typing...